wails-frontend/src/pages/passwordPrompt.tsx

```diff
@@ -53,20 +53,27 @@ function PasswordPrompt() {
     }
   }
 
+  // error state for the input password field
   const [error, setError] = useState<IErrorObject | null>(null);
+  // error state for the input password confirm field
+  const [errorConfirm, setErrorConfirm] = useState<IErrorObject | null>(null);
 
   function validate(e: SyntheticEvent) {
     const form = parseForm(e);
     const { password, passwordConfirm } = form;
 
+    // reset error states
+    setError(null);
+    setErrorConfirm(null);
```

Comment:
Please, double check with @LucasF243 because AFAIK he wants to keep the field in error state (red) until the next submit. At least was agreed in the design system...

<img width="463" alt="Screenshot 2023-05-29 at 15 30 44" src="https://github.com/massalabs/thyra-plugin-wallet/assets/126461030/655a9e11-3ac2-4d42-8c51-afb92521918e">

Reply:
we are in the validate function triggered when user clicks on the submit button. so the error state will not change until the next submit.

Reply:
my question is more: do we show error 'passwords are not the same' bellow the confirm password field even though the password field has error 'must be at least 5 chars'?

Reply:
Let me see if I got the point: you mean more than one error together or one by one?
If is that you are talking about the way Lucas structured that we should go one by one if we have more than one error 😅 . At least for now!

Reply:
@SlnPons prefers to show all errors, and it easily doable: see my last commit

Reply:
I am not against, however I would like to understand how this will looks like in terms of UI/UX.

Reply:
![Uploading image.png…]()

Reply:
Ohh! Beautiful! 👏 

Now I got your point. For the next time, add that under description of the MR.

Thanks for explaining!